### DISCUSSION TITLE
How to ignore the audit of a certain package?

### DISCUSSION THREAD

**Discussion by derekhe** [pnpm/pnpm#5229] [Q&A]
After I use "pnpm audit" and fix the packages, I see some packages are causing issue and I want to keep the older version. 
I don't know how can I ignore auditing the packages. The npm audit tool chain doesn't work for pnpm-lock file.

**Comment by o-martin** [comment]
This will be great! You can use https://github.com/IBM/audit-ci to ignore some alerts until `pnpm` offers such feature
